### PR TITLE
WIP feat: recent trades API

### PR DESCRIPTION
**Pull request by lispc**
1. recent trades API
2. make startall / make viewlogs / make stopall
3. store enum as str in db
3. some fixes of kline
4. some fixes of db

### CODE REVIEW

**Comment by lispc** [comment]
I will add quote_amount and taker_side into TradeRecord, which are useful for recents trades & ticker

**Comment by noel2004** [comment]
> I will add quote_amount and taker_side into TradeRecord, which are useful for recent trades & ticker

I think quote_amount = amount / price? It would be a tradeoff of space and time.

File: Makefile [hunk 1]
```diff
@@ -15,6 +16,20 @@ fmtjs:
 
 fmt: fmtproto fmtsql fmtrs fmtjs
 
+startall:
+	cargo build
```
Comment (ChrisLinn) [inline Makefile:20]: mkdir -p logs

**Comment by lispc** [comment]
> > I will add quote_amount and taker_side into TradeRecord, which are useful for recent trades & ticker
> 
> I think quote_amount = amount / price? It would be a tradeoff of space and time.

quote_amount = amount * price.  Sum of quote amount is widely used, so storing it will be better

**Pull request merged**